docs/releases/2.2.3.txt

```diff
@@ -12,3 +12,6 @@ Bugfixes
 * Fixed a regression in Django 2.2 where :class:`~django.db.models.Avg`,
   :class:`~django.db.models.StdDev`, and :class:`~django.db.models.Variance`
   crash with ``filter`` argument (:ticket:`30542`).
+
+* Fixed a regression in Django 2.2 where using ipdb would prevent the
+  auto-reloader from working (:ticket:`30588`).
```

Comment:
Maybe:
```
* Fixed a regression in Django 2.2.2 where auto-reloader crashes with
  ``AttributeError``, e.g. when using ``ipdb`` (:ticket:`30588`).
```
``ipdb`` can be removed from ``docs/spelling_wordlist``.

Reply:
Done 👍